index.js

```diff
@@ -3,12 +3,12 @@ var _ = require('lodash')
 var Q = require('kew')
 var ipfs = require('ipfs-api')
 var waterfall = require('promise-waterfall')
-var freeport = require('freeport')
-var shutdown = require('shutdown-handler-krlpatch')
+var shutdown = require('shutdown-handler')
 var rimraf = require('rimraf')
 var fs = require('fs')
 
 var IPFS_EXEC = __dirname + '/node_modules/.bin/ipfs'
+var GRACE_PERIOD = 2000 // amount of ms to wait before sigkill
```

Comment:
yeah i think something like 5s or 10s more reasonable. if we can hide things (i.e. the electron-app doesnt hang while closing, but "appears to close" quickly, its fine if it finishes tearing down in the background.